app/models/link_set.rb

```diff
@@ -1,3 +1,3 @@
 class LinkSet < ActiveRecord::Base
-  has_many :links, dependent: :destroy
+  has_many :links, -> { order(id: :asc) }, dependent: :destroy
```

Comment:
It might be better to order by `content_id`, rather than `id` as that means that the order can be communicated to users, as `content_id` is external, and `id` is internal?